Makefile

```diff
@@ -389,7 +389,7 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=v0.7
+protoVer=v0.8
 protoImageName=tendermintdev/sdk-proto-gen:$(protoVer)
```

Comment:
Could someone with access please rebuild the updated image and push it to the docker hub?

Reply:
This is done automatically: https://github.com/cosmos/cosmos-sdk/actions/workflows/proto-docker.yml

Reply:
You shouldn't reverted the version change. What I meant was the docker image will be pushed automatically.